external/YamlCppLib.cmake

```diff
@@ -24,14 +24,17 @@
 if(EXISTS ${REPOSITORY_DIR}/build/ThirdParty/share/yaml-cpp-0.6.2.tar.gz)
     set(URL ${REPOSITORY_DIR}/build/ThirdParty/share/yaml-cpp-0.6.2.tar.gz)
 else()
-    set(URL https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.2.tar.gz)
+    #set(URL https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.2.tar.gz)
+    set(REPO https://github.com/jbeder/yaml-cpp.git)
```

Comment:
is this needed, safe to use master for dependencies?

Reply:
> is this needed, safe to use master for dependencies?

This is temporary.  By using the latest code we get the fix that prevents yaml-cpp from crashing in debug mode.  Also, I was looking carefully at the 0.6.2 download and saw that the changes listed in the release notes were not actually in the code. So I don't know what that file is but it is probably not the real 0.6.2 release.

As soon as they do another release I will switch back.

Reply:
>  By using the latest code we get the fix that prevents yaml-cpp from crashing in debug mode 

Great, just please open an Issue to turn it back to release later. 

>  Also, I was looking carefully at the 0.6.2 download and saw that the changes listed in the release notes were not actually in the code. So I don't know what that file is but it is probably not the real 0.6.2 release. 

Wow! 😮 Would you like to open a bug for their repo?